webapp/app/modules/rooms/rooms.selectors.js

```diff
@@ -0,0 +1,34 @@
+import { createSelector } from 'reselect';
+import { fromJS, Map } from 'immutable';
+import { complement, isNil } from 'ramda';
+
+
+export const selectRooms = (state) => state.get('rooms', Map());
+
+export const selectRoomsList = createSelector(
+  selectRooms,
+  (state) => state.get('roomsList', Map()),
+);
+
+export const selectItems = createSelector(
+  selectRooms,
+  (state) => state.get('items', Map()),
+);
+
+export const selectActiveRoomId = createSelector(
+  selectRooms,
+  (state) => state.get('activeRoomId', ''),
+);
+
+export const selectActiveRoom = createSelector(
+  selectActiveRoomId, selectRoomsList,
+  (activeRoomId, rooms) => (fromJS({
```

Comment:
() wrapujace fromJS sa niepotrzebne

Reply:
Bez tego nie mam `Immutable` tylko zwykły obiekt, wolałbym mieć `Immutable`, czy uważasz, że to niepotrzebne ?